controllers/managers.js

```diff
@@ -140,3 +142,75 @@ exports.login = async (req, res) => {
             });
         });
 };
+
```

Comment:
Thank you very much! I read your code and think it's not he most beautiful but it is the most functional and I am grateful. Very ingenious 👍